[2020-05-07 16:52:17 -0700] [9330] [ERROR] Invalid address: ('gorgonzola.cogsci.uci.edu', 20455)
[2020-05-07 16:52:17 -0700] [9330] [ERROR] Retrying in 1 second.
[2020-05-07 16:52:18 -0700] [9330] [ERROR] Invalid address: ('gorgonzola.cogsci.uci.edu', 20455)
[2020-05-07 16:52:18 -0700] [9330] [ERROR] Retrying in 1 second.
[2020-05-07 16:52:19 -0700] [9330] [ERROR] Invalid address: ('gorgonzola.cogsci.uci.edu', 20455)
[2020-05-07 16:52:19 -0700] [9330] [ERROR] Retrying in 1 second.
[2020-05-07 16:52:20 -0700] [9330] [ERROR] Invalid address: ('gorgonzola.cogsci.uci.edu', 20455)
[2020-05-07 16:52:20 -0700] [9330] [ERROR] Retrying in 1 second.
[2020-05-07 16:52:21 -0700] [9330] [ERROR] Invalid address: ('gorgonzola.cogsci.uci.edu', 20455)
[2020-05-07 16:52:21 -0700] [9330] [ERROR] Retrying in 1 second.
[2020-05-07 16:52:22 -0700] [9330] [ERROR] Can't connect to ('gorgonzola.cogsci.uci.edu', 20455)
2020-08-09 15:06:21,180 Scheduler started
2020-08-09 15:06:21,441 Scheduler started
2020-08-09 15:06:21,703 Scheduler started
2020-08-09 15:06:21,731 Looking for jobs to run
2020-08-09 15:06:21,743 No jobs; waiting until a job is added
2020-08-09 15:06:21,820 Looking for jobs to run
2020-08-09 15:06:21,829 No jobs; waiting until a job is added
2020-08-09 15:06:21,832 Scheduler started
2020-08-09 15:06:21,926 Scheduler started
2020-08-09 15:06:21,966 Scheduler started
2020-08-09 15:06:22,044 Looking for jobs to run
2020-08-09 15:06:22,058 No jobs; waiting until a job is added
2020-08-09 15:06:22,111 Scheduler started
2020-08-09 15:06:22,118 Scheduler started
2020-08-09 15:06:22,126 Scheduler started
2020-08-09 15:06:22,156 Looking for jobs to run
2020-08-09 15:06:22,195 No jobs; waiting until a job is added
2020-08-09 15:06:22,222 Looking for jobs to run
2020-08-09 15:06:22,232 No jobs; waiting until a job is added
2020-08-09 15:06:22,343 Looking for jobs to run
2020-08-09 15:06:22,359 No jobs; waiting until a job is added
2020-08-09 15:06:22,459 Looking for jobs to run
2020-08-09 15:06:22,463 Looking for jobs to run
2020-08-09 15:06:22,486 No jobs; waiting until a job is added
2020-08-09 15:06:22,549 No jobs; waiting until a job is added
2020-08-09 15:06:22,578 Looking for jobs to run
2020-08-09 15:06:22,588 No jobs; waiting until a job is added
[2025-05-02 12:16:20 -0700] [39125] [CRITICAL] WORKER TIMEOUT (pid:39133)
[2025-05-02 12:25:21 -0700] [39125] [CRITICAL] WORKER TIMEOUT (pid:39131)
[2025-05-02 12:26:04 -0700] [39125] [CRITICAL] WORKER TIMEOUT (pid:39132)
[2025-05-02 12:26:49 -0700] [39125] [CRITICAL] WORKER TIMEOUT (pid:39192)
[2025-05-02 12:26:50 -0700] [39125] [CRITICAL] WORKER TIMEOUT (pid:39135)
[2025-05-02 12:27:33 -0700] [39125] [CRITICAL] WORKER TIMEOUT (pid:39134)
[2025-05-02 12:28:29 -0700] [39125] [CRITICAL] WORKER TIMEOUT (pid:39130)
[2025-05-02 12:28:47 -0700] [39125] [CRITICAL] WORKER TIMEOUT (pid:39136)
[2025-05-02 12:34:40 -0700] [39125] [CRITICAL] WORKER TIMEOUT (pid:39194)
[2025-05-02 12:35:38 -0700] [39125] [CRITICAL] WORKER TIMEOUT (pid:39190)
[2025-05-02 12:36:58 -0700] [39125] [CRITICAL] WORKER TIMEOUT (pid:39129)
[2025-05-02 12:37:19 -0700] [39125] [CRITICAL] WORKER TIMEOUT (pid:39195)
[2025-05-02 12:37:21 -0700] [39125] [CRITICAL] WORKER TIMEOUT (pid:39128)
[2025-05-02 12:38:42 -0700] [39125] [CRITICAL] WORKER TIMEOUT (pid:39204)
[2025-05-02 12:38:42 -0700] [39125] [CRITICAL] WORKER TIMEOUT (pid:39205)
[2025-05-02 12:39:41 -0700] [39125] [CRITICAL] WORKER TIMEOUT (pid:39209)
[2025-05-02 12:39:41 -0700] [39125] [CRITICAL] WORKER TIMEOUT (pid:39210)
[2025-05-02 12:40:07 -0700] [39125] [CRITICAL] WORKER TIMEOUT (pid:39213)
[2025-05-02 12:44:21 -0700] [39125] [CRITICAL] WORKER TIMEOUT (pid:39199)
[2025-05-02 12:44:21 -0700] [39125] [CRITICAL] WORKER TIMEOUT (pid:39217)
[2025-05-02 12:48:22 -0700] [39125] [CRITICAL] WORKER TIMEOUT (pid:39218)
[2025-05-02 13:03:34 -0700] [39125] [CRITICAL] WORKER TIMEOUT (pid:39216)
[2025-05-02 13:09:23 -0700] [39125] [CRITICAL] WORKER TIMEOUT (pid:39193)
2025-05-08 16:55:13,570 Exception on /.well-known/gpc.json [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2311, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1834, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1737, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 36, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1832, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1818, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/psiturk/Library/Python/3.7/lib/python/site-packages/psiturk/experiment.py", line 749, in regularpage
    return render_template(foldername+"/"+pagename)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: .well-known/gpc.json
2025-05-08 16:55:18,229 Exception on /.well-known/gpc.json [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2311, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1834, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1737, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 36, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1832, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1818, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/psiturk/Library/Python/3.7/lib/python/site-packages/psiturk/experiment.py", line 749, in regularpage
    return render_template(foldername+"/"+pagename)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: .well-known/gpc.json
2025-05-08 16:55:34,201 Exception on /.well-known/gpc.json [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2311, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1834, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1737, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 36, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1832, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1818, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/psiturk/Library/Python/3.7/lib/python/site-packages/psiturk/experiment.py", line 749, in regularpage
    return render_template(foldername+"/"+pagename)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: .well-known/gpc.json
2025-05-08 16:55:41,895 Exception on /.well-known/gpc.json [GET]
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 2311, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1834, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1737, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python3.7/site-packages/flask/_compat.py", line 36, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1832, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python3.7/site-packages/flask/app.py", line 1818, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/psiturk/Library/Python/3.7/lib/python/site-packages/psiturk/experiment.py", line 749, in regularpage
    return render_template(foldername+"/"+pagename)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 134, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/usr/local/lib/python3.7/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/usr/local/lib/python3.7/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 58, in get_source
    return self._get_source_fast(environment, template)
  File "/usr/local/lib/python3.7/site-packages/flask/templating.py", line 86, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: .well-known/gpc.json
[2025-05-08 16:59:05 -0700] [46289] [CRITICAL] WORKER TIMEOUT (pid:46296)
2025-05-08 17:06:24,379 browser_type_not_allowed (1014) {'assignmentId': 'debugz8rYz', 'hitId': 'debugoP2Tx', 'workerId': 'debugeMziT', 'mode': 'debug'}
[2025-05-08 17:17:03 -0700] [46289] [CRITICAL] WORKER TIMEOUT (pid:46300)
[2025-05-08 17:17:05 -0700] [46289] [CRITICAL] WORKER TIMEOUT (pid:46295)
[2025-05-08 18:06:22 -0700] [46289] [CRITICAL] WORKER TIMEOUT (pid:46294)
[2025-05-08 19:19:51 -0700] [46289] [CRITICAL] WORKER TIMEOUT (pid:46314)
[2025-05-08 19:19:51 -0700] [46289] [CRITICAL] WORKER TIMEOUT (pid:46320)
[2025-05-08 19:19:51 -0700] [46289] [CRITICAL] WORKER TIMEOUT (pid:46322)
[2025-05-08 19:20:00 -0700] [46289] [CRITICAL] WORKER TIMEOUT (pid:46297)
[2025-05-08 19:20:15 -0700] [46289] [CRITICAL] WORKER TIMEOUT (pid:46383)
[2025-05-08 19:20:17 -0700] [46289] [CRITICAL] WORKER TIMEOUT (pid:46293)
[2025-05-08 19:21:22 -0700] [46289] [CRITICAL] WORKER TIMEOUT (pid:46298)
[2025-05-08 19:21:23 -0700] [46289] [CRITICAL] WORKER TIMEOUT (pid:46425)
[2025-05-08 19:48:52 -0700] [46289] [CRITICAL] WORKER TIMEOUT (pid:46424)
[2025-05-08 19:48:54 -0700] [46289] [CRITICAL] WORKER TIMEOUT (pid:46423)
[2025-05-08 19:51:52 -0700] [46289] [CRITICAL] WORKER TIMEOUT (pid:46292)
[2025-05-08 19:53:43 -0700] [46289] [CRITICAL] WORKER TIMEOUT (pid:46477)
[2025-05-08 19:56:08 -0700] [46289] [CRITICAL] WORKER TIMEOUT (pid:46475)
[2025-05-08 19:57:54 -0700] [46289] [CRITICAL] WORKER TIMEOUT (pid:46479)
[2025-05-08 19:58:10 -0700] [46289] [CRITICAL] WORKER TIMEOUT (pid:46431)
[2025-05-08 19:58:12 -0700] [46289] [CRITICAL] WORKER TIMEOUT (pid:46474)
[2025-05-08 20:00:37 -0700] [46289] [CRITICAL] WORKER TIMEOUT (pid:46427)
[2025-05-08 20:01:36 -0700] [46289] [CRITICAL] WORKER TIMEOUT (pid:46429)
[2025-05-08 20:01:46 -0700] [46289] [CRITICAL] WORKER TIMEOUT (pid:46482)
[2025-05-08 20:05:03 -0700] [46289] [CRITICAL] WORKER TIMEOUT (pid:46428)
[2025-05-08 21:00:22 -0700] [46289] [CRITICAL] WORKER TIMEOUT (pid:46487)
[2025-05-08 21:02:05 -0700] [46289] [CRITICAL] WORKER TIMEOUT (pid:46478)
[2025-05-08 21:22:12 -0700] [46289] [CRITICAL] WORKER TIMEOUT (pid:46481)
2025-05-09 19:01:58,846 hit_assign_worker_id_not_set_in_consent (1002) {}
2025-05-09 19:02:09,762 hit_assign_worker_id_not_set_in_mturk (1001) {}
2025-05-09 20:28:18,136 hit_assign_worker_id_not_set_in_exp (1003) {}
